site/docs/foundations/spacing.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Spacing
+layout: DetailTechnical
+sidebar:
+  label: Spacing
+data:
+  status: In progress
+---
+
+The organization of space is a key aspect of design. Spatial tokens help align layouts and create consistent visual rhythm.
+
+The spacing system is structured around a base unit for each density (`spacing-100`). This determines the spacing ramp for each density across the design system and helps create a consistent visual aesthetic.
+
+## Spacing scale
```

Comment:
are we missing size? @pseys

Reply:
Yes. I was thinking size makes more sense to pair with scale, so was planning to either add it to Scale or create a Size foundation.